component-catalog-app/Examples/Tabs.elm

```diff
@@ -287,20 +293,31 @@ initSettings =
                 , ( "Right", Control.value Right )
                 ]
             )
-        |> Control.field "customSpacing"
+        |> Control.field "customSpacing" (Control.maybe False (values [ 2, 3, 4, 8, 16 ]))
+        |> Control.field "withTooltips" (Control.bool True)
+        |> Control.field "tabListBackgroundColor" (Control.maybe False colorChoices)
+        |> Control.field "highContrastTabListBackgroundColor" (Control.maybe False colorChoices)
+        |> Control.field "tabListSticky"
             (Control.maybe False
                 (Control.choice
-                    [ ( "2", Control.value 2 )
-                    , ( "3", Control.value 3 )
-                    , ( "4", Control.value 4 )
-                    , ( "8", Control.value 8 )
-                    , ( "16", Control.value 16 )
+                    [ ( "Default", Control.value Default )
+                    , ( "Custom"
+                      , Control.record Tabs.TabListStickyConfig
+                            |> Control.field "topOffset" (values [ 0, 10, 50 ])
+                            |> Control.field "zIndex" (values [ 0, 1, 5, 10 ])
+                            |> Control.field "includeMobile" (Control.bool False)
+                            |> Control.map Custom
+                      )
                     ]
                 )
             )
-        |> Control.field "withTooltips" (Control.bool True)
-        |> Control.field "tabListBackgroundColor" (Control.maybe False colorChoices)
-        |> Control.field "highContrastTabListBackgroundColor" (Control.maybe False colorChoices)
+
+
+values : List a -> Control a
+values nums =
+    nums
+        |> List.map (\n -> ( Debug.toString n, Control.value n ))
+        |> Control.choice
```

Comment:
Could you make this more specific (so it doesn't use Debug.toString) and move it to `Debug.Control.Extra` where the other extensions to Debug.Control live? Thanks!